docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/payara-server.adoc

```diff
@@ -0,0 +1,53 @@
+= Payara Server tools in IntelliJ
+
+[[deploy-maven-application-to-payara-server]]
+== Deploy the Maven Web Application to Payara Server
+
+* To deploy the Maven application to Payara Enterprise Server, first create the new *Payara 
+Server configuration* by selecting the *Edit Configuration* option:
+
+image::intellij-plugin/payara-server/create-configuration.png[Create Configuration]
+
+* In the *Run/Debug Configurations* dialog, click *+*, expand the *Payara Server* 
+node, and select *Local*:
+
+image::intellij-plugin/payara-server/create-local-server-configuration.png[Create Local Server configuration]
+
+* To add the Payara Server to the configuration, click the *Configure...* button 
+and browse the Payara Server installation path:
+
+image::intellij-plugin/payara-server/browse-payara-server.png[Browse the Payara Server path]
+
+* Fix the artifacts warnings that appear at the bottom of the run configuration 
+settings dialog, by clicking the *Fix* button:
+
+image::intellij-plugin/payara-server/add-artifact.png[Add artifact]
+
+* Select the *sample-micro-app:war* artifact for deployment to the Payara Enterprise Server:
+
+image::intellij-plugin/payara-server/select-war-artifact.png[Select WAR artifact]
+
+* Now fix the debugger related warning by pressing the *Fix* button, which updates 
+the debugger settings in the domains.xml of the selected Payara Enterprise Server�s domain:
```

Comment:
```suggestion
the debugger settings in the domains.xml of the selected Payara Enterprise Server's domain:
```
This is subtle, but the docs will render the existing apostrophe as an unknown character. It's probably not in the correct encoding.